.gitmodules

```diff
@@ -66,3 +66,6 @@
 [submodule "src/linker"]
 	path = src/linker
 	url = https://github.com/mono/linker.git
+[submodule "src/aspnet-razor"]
+	path = src/aspnet-razor
+	url = https://github.com/aspnet/Razor
```

Comment:
```suggestion
	url = https://github.com/aspnet/Razor
	branch = release/2.1
```

Reply:
AFAIK the only thing this allows is `git submodule update --remote`, and I don't think anyone uses that for source-build. From an infra perspective this is just another value to maintain and resolve conflicts for.

If someone made a case for adding these (to all the submodules that have applicable branches), I could get behind that, but for now I want to leave it out.